src/getEntityTypescriptPostgres.ts

```diff
@@ -80,14 +93,14 @@ function build_column({
 		current_col += " = " + defaultVal;
 	}
 
-	return current_col;
+	return { col: current_col, decorator_fn, decorator_import };
 }
 
-// TODO: everything optionnal
 export async function getEntitiesTypescriptPostgres(
```

Comment:
![image](https://github.com/Yedidyar/remult-cli/assets/42849252/69b16a86-e777-4938-88b0-4cd61dd14010)

Reply:
What is this message? 
I don't have this :o

Reply:
It coms from Eslint

Reply:
I don't have it, and running lint doesn't conplain.

![image](https://github.com/Yedidyar/remult-cli/assets/5312607/3ff2867d-ac6e-4e92-b806-ff6e0b6dcef6)


I'm missing something?

Reply:
I dont know why you can't see it

Reply:
Because you have a sonar plugin in your IDE?
You have no issue running the lint command as well I guess?

Reply:
no i disable it and it still errors i add it to ci but it like your machine maybe it linux?

Reply:
I'm on windows, running WSL (linux).

When you do: `pnpm lint`, you have an error?